### PR TITLE
chore: Refactor mocked access tokens

### PR DESCRIPTION
**Pull request by marikaner**
Reduce the number of predefined tokens as they were repeated and often times unnecessary.

### CODE REVIEW

File: packages/connectivity/src/scp-cf/destination/destination-accessor-failure-cases.spec.ts [hunk 1]
```diff
@@ -37,19 +36,17 @@ describe('Failure cases', () => {
...
       getDestination({
         destinationName,
-        jwt: subscriberServiceTokenWithVerificationURL,
+        jwt: subscriberServiceToken,
         cacheVerificationKeys: false,
         iasToXsuaaTokenExchange: false
       })
     ).rejects.toThrowErrorMatchingInlineSnapshot(
-      '"Could not find service binding for type \'destination\'."'
+      '"Could not find binding to service \'destination\', that includes credentials."'
```
Comment (marikaner) [inline packages/connectivity/src/scp-cf/destination/destination-accessor-failure-cases.spec.ts:49]:
```suggestion
      '"Could not find service binding for type \'destination\'."'
```

File: packages/connectivity/src/scp-cf/jwt.spec.ts [hunk 2]
```diff
@@ -41,36 +35,6 @@ export function responseWithPublicKey(key = publicKey) {
 }
 
 describe('jwt', () => {
```
Comment (marikaner) [inline packages/connectivity/src/scp-cf/jwt.spec.ts:37]:
```suggestion
describe('jwt', () => {
describe('isXsuaa()', () => {
    it('returns true if jku and uaa are from same domain', () => {
      const jwt = decodeJwtComplete(
        signedJwtForVerification(
          {},
          'https://myTenant.authentication.sap.hana.ondemand.com/token_keys'
        )
      );
      mockServiceBindings();
      expect(isXsuaaToken(jwt)).toBe(true);
    });

    it('returns false if jku is missing', () => {
      const jwt = decodeJwtComplete(signedJwtForVerification({}, undefined));
      mockServiceBindings();
      expect(isXsuaaToken(jwt)).toBe(false);
    });

    it('returns false if jku and uaa are from different domain', () => {
      const jwt = decodeJwtComplete(
        signedJwtForVerification(
          {},
          'https://myTenant.some.non.xsuaa.domain.com/token_keys'
        )
      );
      mockServiceBindings();
      expect(isXsuaaToken(jwt)).toBe(false);
    });
  });
  
```

**Review by deekshas8 — APPROVED** [review]
LGTM. Just a minor question.

File: packages/connectivity/src/scp-cf/connectivity-service.spec.ts [hunk 3]
```diff
@@ -69,14 +69,14 @@ describe('connectivity-service', () => {
...
-          'SAP-Connectivity-Authentication': `Bearer ${providerUserJwt}`
+          'SAP-Connectivity-Authentication': `Bearer ${providerUserToken}`
         }
       }
     };
     const withProxy = await addProxyConfigurationOnPrem(
       input,
       getRequiredSubscriberToken({
-        userJwt: getJwtPair(providerUserJwt)
+        userJwt: getJwtPair(providerUserToken)
```
[the remark this answers is not in the document]
Comment (marikaner) [reply packages/connectivity/src/scp-cf/connectivity-service.spec.ts:79]: I did not change this test (just renamed the variable), but I think it doesn't matter for this one, does it?

**Pull request merged**